### PR TITLE
Forward original event on drag

### PR DESCRIPTION
**Pull request by yohanboniface**
This PR adds forwarding of the original event when firing `drag` or `move` events.

Use case: handle ctrlKey/altKey, etc. while dragging for creating different UX scenarios (eg. do not snap if `ctrlKey` is pressed while dragging).

Thanks for reviewing :)

### CODE REVIEW

File: src/dom/Draggable.js [hunk 1]
```diff
@@ -99,13 +99,16 @@ L.Draggable = L.Evented.extend({
 		this._moving = true;
 
 		L.Util.cancelAnimFrame(this._animRequest);
-		this._animRequest = L.Util.requestAnimFrame(this._updatePosition, this, true, this._dragStartTarget);
+		var animationCallback = function () {
+			this._updatePosition(e);
+		};
+		this._animRequest = L.Util.requestAnimFrame(animationCallback, this, true, this._dragStartTarget);
```
Comment (mourner) [inline src/dom/Draggable.js]: You can use `L.bind(animationCallback, null, e)` instead. Or `this._lastEvent` to avoid closure entirely.

Comment (yohanboniface) [inline src/dom/Draggable.js]: Good point!
Which one do you prefer?

File: src/layer/marker/Marker.Drag.js [hunk 2]
```diff
@@ -58,10 +58,11 @@ L.Handler.MarkerDrag = L.Handler.extend({
 		}
 
 		marker._latlng = latlng;
+		e.latlng = latlng;
 
 		marker
-		    .fire('move', {latlng: latlng})
-		    .fire('drag');
+		    .fire('move', e)
```
Comment (mourner) [inline src/layer/marker/Marker.Drag.js]: I don't like modified DOM event being passed as event data... In a similar situation in other place of Leaflet code, original event is passed as `e.originalEvent`.

File: src/dom/Draggable.js (hunk 1 above)
Comment (yohanboniface) [inline src/dom/Draggable.js]: Latest (`this._lastEvent`) may be more readable, given that we are already passing a buntch of argument to `requestAnimFrame`.

File: src/layer/marker/Marker.Drag.js (hunk 2 above)
Comment (yohanboniface) [inline src/layer/marker/Marker.Drag.js]: I originally done that, but then changed my mind, finding this version lighter, but I'll do that then :)

Comment (yohanboniface) [inline src/layer/marker/Marker.Drag.js]: Also, as we are doing two forwards, I'm not sure at which step we should introduce the `originalEvent` key ;)

Comment (mourner) [inline src/layer/marker/Marker.Drag.js]: Probably at the start.

**Comment by yohanboniface** [comment]
Anything more I need to clean? :)

**Pull request merged**